### ISSUE TITLE
Holidays dont load correctly 

### ISSUE THREAD

**Issue by kaifresh** [commenthol/date-holidays#91]
For holidays in Australia, instantiating holidays using an object doesn't work. 

For example the following fails, creating an instance with an empty `{}` instead of a set of holidays

```javascript
const holidays = new Holidays({country: 'AU', state: 'NSW'});
```

Whereas this creates holidays as expected

```javascript            
const holidays = new Holidays('AU', 'NSW');
```

**Comment by commenthol** [comment]
Issue confirmed.

**Comment by commenthol** [comment]
Is fixed in date-holidays-parser@1.3.2 with https://github.com/commenthol/date-holidays-parser/pull/10
Thanks for rising this issue.
